src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayCost = 40;
+  const total = days * dayCost;
+  let discount;
+
+  if (days < 3) {
+    return total;
+  } else if (days < 7) {
+    discount = 20;
+
+    return total - discount;
+  } else {
+    discount = 50;
```

Comment:
basically declaring with `let` and then assign a value is fine, yet in this case you just have two options,
so better declare them right away and use inside return statements - saves few lines of code
and when you have `if () { return` return terminates function execution, so no actual need to nest `else if / else`:
```suggestion
  }

  if (days < 7) {
    return total - discountTwenty;
  }

  return total - discountFifty
```